From 9e0e316409db50ffd0f13199ccbc9e306cfc01b5 Mon Sep 17 00:00:00 2001
From: xplodwild <xplod@ouverta.fr>
Date: Sun, 25 Aug 2013 18:23:57 +0100
Subject: [PATCH 3/7] Makefile: Fix target display

Change-Id: I07e60e6b9a6b09d798ca50c6295072828b643a50
---
 core/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/Makefile b/core/Makefile
index 3e140f7..cab526d 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1587,7 +1587,7 @@ otapackage: $(INTERNAL_OTA_PACKAGE_TARGET)
 bacon: otapackage
 	$(hide) ln -f $(INTERNAL_OTA_PACKAGE_TARGET) $(CUSTOM_TARGET_PACKAGE)
 	$(hide) $(MD5SUM) $(CUSTOM_TARGET_PACKAGE) > $(CUSTOM_TARGET_PACKAGE).md5sum
-	@echo -e "Package complete: $(CM_TARGET_PACKAGE)"
+	@echo -e "Package complete: $(CUSTOM_TARGET_PACKAGE)"
 
 endif    # recovery_fstab is defined
 endif    # TARGET_NO_KERNEL != true
-- 
1.9.1

