From 943aca2e39a172ee67cbf39bcc621f7da51a19de Mon Sep 17 00:00:00 2001
From: Erik Castricum <erikcas1972@gmail.com>
Date: Wed, 3 Jun 2015 14:42:21 +0200
Subject: [PATCH 6/7] Update Makefile

---
 core/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/Makefile b/core/Makefile
index 3e6d405..47afeac 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -863,6 +863,7 @@ $(TARGET_RECOVERY_ROOT_TIMESTAMP): $(INTERNAL_RECOVERY_FILES) \
 		$(INSTALLED_BOOTIMAGE_TARGET) \
 		$(recovery_binary) \
 		$(recovery_initrc) \
+		$(recovery_initrc) $(recovery_sepolicy) $(recovery_kernel) \
 		$(INSTALLED_2NDBOOTLOADER_TARGET) \
 		$(recovery_build_prop) $(recovery_resource_deps) \
 		$(recovery_fstab) \
-- 
1.9.1

